qualtran/bloqs/chemistry/thc/select_bloq.py

```diff
@@ -193,6 +194,7 @@ def build_composite_bloq(
         plus_b: SoquetT,
         sys_a: SoquetT,
         sys_b: SoquetT,
+        **kwargs,
```

Comment:
what is this? document

Reply:
Fixing the Select / Prepare interfaces and their implementations for THC is one of the last two things of this PR that needs to be extracted out (other being phase estimation bloq). This PR "makes things work" in a hacky way and it's on my TODO list to fix them the right way. Here is a related issue https://github.com/quantumlib/Qualtran/issues/549

Reply:
I've opened a new issue for this - https://github.com/quantumlib/Qualtran/issues/841